src/commands/settings/command/toggle.ts

```diff
@@ -34,9 +34,10 @@ export default class Toggle {
     const commandPath = interaction.options.getString("command", true);
     let [commandName, subcommandName, subcommandGroupName] = commandPath.split(" ");
     commandName = commandName.replace("/", "");
-    const disabledCommands = await settingsTable?.get(`${interaction.guild.id}.disabledCommands`).then(
-      (disabledCommands) => disabledCommands as any[] ?? []
-    ).catch(() => []);
+    const disabledCommands = await settingsTable
+      ?.get(`${interaction.guild.id}.disabledCommands`)
```

Comment:
```suggestion
    const disabledCommands = await settingsTable?
      .get(`${interaction.guild.id}.disabledCommands`)
```
That character belongs there I think ;)